src/pint/scripts/pint_parameter.py

```diff
@@ -0,0 +1,221 @@
+""" A script that quarry the PINT built-in parameters. Currently this script
+only has a limited functionality (Print all parameters in .rst format).
+"""
+
+import argparse
+import copy
+from collections import OrderedDict
+from pint.models.timing_model import TimingModel, Component
+from pint.utils import get_param_name_map, split_prefixed_name, PrefixError
+
+
+def get_request_info(par, info):
+    """ Get the requested infor from the parameter object.
+
+    Parameter
+    ---------
+    par: `pint.models.Parameter` object
+        The parameter to get information from
+    info: list of strings
+        The request list of information. They should be the parametre attribute
+        name.
+    """
+    result = {"name": par.name, "type": type(par).__name__}
+    for key in request_info:
+        result[key] = getattr(par, key)
+    return result
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(
+        description="Output the all parameters in"
+        "the built-in models."
+        "The current output is"
+        "['name', 'description',"
+        " 'unit', 'value', 'aliases']"
+    )
+    parser.add_argument(
+        "-p",
+        "--parameter",
+        nargs="*",
+        default=["all"],
+        help="Quarried parameter names.",
```

Comment:
Er, does this allow requesting multiple specific parameters? Comma-separated?

Reply:
This allows `-p F1 F0 DEC` type of input.